### PR TITLE
Add courier package tracking app

### LINKED ISSUE #50: [Proposal] Courier Package Tracking
# Summary

An instant app to track your package using the tracking id. 

# Example search queries

- FedEx tracking
- Delhivery tracking

# Implementation (optional)

Some of the courier providers have documentation for their APIs, like Delhivery  and FedEx. 

# Context

### PR DESCRIPTION
**Pull request by DefCon-007**
- Use Pickrr API to get courier status
- Add lodash package
- Fixes #50

Screenshot 2021-10-03 at 8 10 45 PM

### CODE REVIEW

**Comment by DefCon-007** [comment]
Example search query: `current status of 8695542668`

File: src/apps/courierTracking.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
+import styled from "styled-components";
+import { isTriggered } from "../lib/utilityApis";
+import { InstantAppProps, queryToDataInput, queryToDataOutput } from "./types";
+
+const _ = require("lodash");
```
Comment (OrkoHunter) [inline src/apps/courierTracking.tsx:5]:
```suggestion
import _ from 'lodash'
```

Mixed import/require statements 😄

File: src/apps/courierTracking.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    })
+  );
+  results = _.compact(results);
+
+  return results[0];
+}
+
+const CourierTracking = {
+  name: "Courier Tracking",
+  description: "I am a template app, please change this description",
```
Comment (OrkoHunter) [inline src/apps/courierTracking.tsx:188]:
```suggestion
  description: "View any courier package status using Pickrr API",
```

**Review by OrkoHunter — COMMENTED** [review]
Looks awesome! Thanks @DefCon-007 🚀 

A bit of refactor 🚧  going on in https://github.com/felvin-search/instant-apps/pull/128. So, in a day or two, we'll refactor this app in your PR according to the new package structure and merge it! 💯

File: src/apps/courierTracking.tsx (hunk 2 above)
Comment (DefCon-007) [reply src/apps/courierTracking.tsx:188]: Oops, missed this :P 
Thanks for the change :)

**Comment by DefCon-007** [comment]
> A bit of refactor 🚧 going on in #128. So, in a day or two, we'll refactor this app in your PR according to the new package structure and merge it! 💯

Thats pretty big refactor :P

**Review by sahil-shubham — APPROVED** [review]
Fixed the merge issues, this looks good to me. Thanks a lot for working on this @DefCon-007 

We can make issues to improve this for queries that don't contain the tracking ID, i.e. for the query "track courier" we can just show an input box.

**Pull request merged**